component/file_cache/file_cache.go

```diff
@@ -102,7 +102,7 @@ type FileCacheOptions struct {
 
 	CreateEmptyFile bool `config:"create-empty-file" yaml:"create-empty-file,omitempty"`
 	AllowNonEmpty   bool `config:"allow-non-empty-temp" yaml:"allow-non-empty-temp,omitempty"`
-	CleanupOnStart  bool `config:"cleanup-on-start" yaml:"cleanup-on-start,omitempty"`
+	// CleanupOnStart has been removed in favor of the global cleanup-on-start flag in mount.go
```

Comment:
Some existing customers might be using this feature, so leave it in the config and cli param but cleanup directory code can be removed from this file.